controllers/datadogagent/feature/clusterchecks/feature.go

```diff
@@ -57,22 +57,23 @@ func (f *clusterChecksFeature) ID() feature.IDType {
 	return feature.ClusterChecksIDType
 }
 
-func (f *clusterChecksFeature) Configure(dda *v2alpha1.DatadogAgent) feature.RequiredComponents {
+func (f *clusterChecksFeature) Configure(dda *v2alpha1.DatadogAgent) (reqComp feature.RequiredComponents) {
 	clusterChecksEnabled := apiutils.BoolValue(dda.Spec.Features.ClusterChecks.Enabled)
-	f.useClusterCheckRunners = clusterChecksEnabled && apiutils.BoolValue(dda.Spec.Features.ClusterChecks.UseClusterChecksRunners)
-
-	{
-		hash, err := comparison.GenerateMD5ForSpec(dda.Spec.Features.ClusterChecks)
-		if err != nil {
-			f.logger.Error(err, "couldn't generate hash for cluster checks config")
-		} else {
-			f.logger.V(2).Info("created cluster checks", "hash", hash)
-		}
-		f.customConfigAnnotationValue = hash
-		f.customConfigAnnotationKey = object.GetChecksumAnnotationKey(feature.ClusterChecksIDType)
-	}
 
 	if clusterChecksEnabled {
+		f.useClusterCheckRunners = clusterChecksEnabled && apiutils.BoolValue(dda.Spec.Features.ClusterChecks.UseClusterChecksRunners)
```

Comment:
```suggestion
		f.useClusterCheckRunners = apiutils.BoolValue(dda.Spec.Features.ClusterChecks.UseClusterChecksRunners)
```

in fact, maybe we can get rid of the `clusterChecksEnabled` variable and just do `if apiutils.BoolValue(dda.Spec.Features.ClusterChecks.Enabled)` at the top of the block, since we don't need this var elsewhere?

Reply:
I overlooked the redundancy, thanks for catching it, fixed now. Had to force push after rebasing main.